shared/scss/_site-rating-icons.scss

```diff
@@ -1,155 +0,0 @@
-            }
-        }
-    }
-
-    &.site-info__rating--b {
-
-        .is-browser--chrome & {
-            background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__b});
-        }
-
-        .is-browser--moz & {
-            background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__b});
-        }
-
-        &.is-active {
-
-            .is-browser--chrome & {
-                background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__b--is-active});
-            }
-
-            .is-browser--moz & {
-                background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__b--is-active});
-            }
-        }
-    }
-
-    &.site-info__rating--c {
-
-        .is-browser--chrome & {
-            background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__c});
-        }
-
-        .is-browser--moz & {
-            background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__c});
-        }
-
-        &.is-active {
-
-            .is-browser--chrome & {
-                background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__c--is-active});
-            }
-
-            .is-browser--moz & {
-                background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__c--is-active});
-            }
-        }
-    }
-
-    &.site-info__rating--d {
-
-        .is-browser--chrome & {
-            background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__d});
-        }
-
-        .is-browser--moz & {
-            background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__d});
-        }
-
-        &.is-active {
-
-            .is-browser--chrome & {
-                background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__d--is-active});
-            }
-
-            .is-browser--moz & {
-                background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__d--is-active});
-            }
-        }
-    }
-
-    &.site-info__rating--f {
-
-        .is-browser--chrome & {
-            background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__f});
-        }
-
-        .is-browser--moz & {
-            background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__f});
-        }
-
-        &.is-active {
-
-            .is-browser--chrome & {
-                background-image: url(chrome-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__f--is-active});
-            }
-
-            .is-browser--moz & {
-                background-image: url(moz-extension://__MSG_@@extension_id__/img/#{$icon__hero_rating__f--is-active});
-            }
-        }
-    }
-}
```

Comment:
Turns out these just weren't used after we abstracted away the rating hero stuff into a separate template.